clients/lsp-pylsp.el

```diff
@@ -370,6 +375,7 @@ So it will rename only references it can find."
    ("pylsp.plugins.rope_rename.enabled" (lambda () (eq lsp-pylsp-rename-backend 'rope)) t)
    ("pylsp.plugins.autopep8.enabled" lsp-pylsp-plugins-autopep8-enabled t)
    ("pylsp.plugins.yapf.enabled" lsp-pylsp-plugins-yapf-enabled t)
+   ("pylsp.plugins.black.enabled" lsp-pylsp-plugins-black-enabled nil)
```

Comment:
if it is boolean, the last param should be `t`, not nil.

Reply:
fixed in new commit